docs/advanced_usage.md

```diff
@@ -501,3 +501,8 @@ You can also set the default for new tables (this does NOT affect
 existing tables) with:
 
 `ALTER DATABASE <db_name> CHARACTER SET utf8mb4 COLLATE utf8mb4_bin;`
+
+!!! warning
+
+    Some systems may require the use of the `utf8mb3` character set to avoid
```

Comment:
If the user uses mb3, they might have issues consuming some documents.  It's trading maybe one problem for another, which isn't ideal.  Maybe we recommend 10.4+ instead?  Or make the user aware setting this may cause problems with some documents

Reply:
Debian actually [sets the default to utf8mb4](https://mariadb.com/kb/en/differences-in-mariadb-in-debian-and-ubuntu/)

Reply:
Yea good point, that's why I didn't change the default per se. You think we just add that info to the note, like pros / cons?

Reply:
I think we should recommend a default version of 10.4+, with some sort of note that older versions may work, but not guaranteed.

10.3 will only be maintained through May '23 anyway. And Debian Bullseye provides 10.5 in its repos, and was released August of '21, so it's not hard to get it with a somewhat recent OS

Reply:
Ok I reworded this, lmk if makes sense, was a little weird to write in a compact way.